README.md

```diff
@@ -29,6 +29,7 @@ also try,
     $ make check
     $ make install
     $ make install PREFIX=/usr
+    $ make SD_SOCKET_ACTIVATION=true
```

Comment:
This name is a little cryptic to put it into readme without explanation. Advanced systemd users might guess its meaning, but mere mortals won't.

Reply:
@ysmolsky agreed. Thoughts on renaming it to `SYSTEMD_SOCKET_ACTIVATION`?

Reply:
That is better. Please, also put a sentence explaining it.